### DISCUSSION TITLE
[Question][Hw5] pp error

### DISCUSSION THREAD

**Discussion by kws1207** [prosyslab-classroom/cs348-information-security#624] [Q&A]
While debugging, I put the following: 

`let _ = Memory.pp Format.std_formatter fc in`

But the following error occurs: 
```
Cannot find lhs of i32 %div4
Fatal error: exception Not_found
```

I don't know any other debugging tool rather than pp. In such case, how can I debug?

**Comment by RiceBiscuits** [comment]
I recommend you to find which timing and which variable invoke an exceptions.
